jax/numpy/lax_numpy.py

```diff
@@ -3174,6 +3174,28 @@ def unique(ar, return_index=False, return_inverse=False,
   raise NotImplementedError(
         "np.unique is not implemented for the axis argument")
 
+  
+@_wraps(onp.setxor1d)
+def setxor1d(ar1, ar2, assume_unique=False):
+  if not assume_unique:
+    ar1 = unique(ar1)
+    ar2 = unique(ar2)
+  else:
+    if ar1.ndim == 0:
+      ar1 = array([ar1])
+    if ar2.ndim == 0:
+      ar2 = array([ar2])
+  
+  aux = concatenate((ar1, ar2))
+  if aux.size == 0:
+      return aux
+
+  aux.sort()
```

Comment:
This relies on the in-place behavior of NumPy `sort`; you want to use something like `aux = sort(aux)` for JAX compatibility.